nix-mode.el

```diff
@@ -4,7 +4,7 @@
 ;; Homepage: https://github.com/NixOS/nix-mode
 ;; Version: 1.4.4
 ;; Keywords: nix, languages, tools, unix
-;; Package-Requires: ((emacs "25"))
+;; Package-Requires: ((emacs "25.1"))
```

Comment:
I'm not sure if this is right. Isn't there an Emacs 25.0 release? /cc @purcell

Reply:
Nope, the first stable release in a series is always x.1, and this particular change is probably because my `package-lint.el` explicitly recommends depending on the specific release in which the code's relied-upon features became available. IOW this change is good.

Reply:
Incidentally, if you install `package-lint` in the CI setup, perhaps using `emacsWithPackages`,  you can use its batch function to check the file for packaging issues.